tdrs-backend/tdpservice/parsers/schema_defs/tanf/t3.py

```diff
@@ -318,18 +320,20 @@
     ],
 )
 
+
 child_two = RowSchema(
     record_type="T3",
     document=TANF_T3DataSubmissionDocument(),
-    quiet_preparser_errors=True,
+    quiet_preparser_errors=validators.is_quiet_preparser_errors(min_length=101),
     preparsing_validators=[
-        validators.notEmpty(start=60, end=101),
+        validators.t3_m3_child_validator(SECOND_CHILD),
```

Comment:
@raftmsohani i tested the record below which has 2 children and recordlen == 97:

`T320201011111111157120190527WTTTTT9WT12212122204398100000000420100125WTTTT9@TB1221222220430490000`

parsed the first child but not the second, which is expected but no error message was raised. is this expected?

Reply:
no, there was an error with quite_preparsing function which is fixed and should take care of this issue